.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,67 @@
+
+## Your first ~two months
+
+The goal of onboarding is to set you up for success as a new engineering manager at Artsy. To that end, your main objective during this time will be to gather knowledge. You will be given ample time to get acquainted with your new peers, our business, and our tech.
+
+_For now we’ll try to target roughly two months, as there are currently 5 product teams and sprints have a duration of 2 weeks. Whether or not this duration was too arbitrary is something we’ll have to learn._
+
+To help work on the main objective, we’ve tried to identify the following goals.
+
+### Get to know Artsy's team
+
+- [ ] Have 1:1s with each and every engineer (that includes tech-leads) to get to know the people, their interests, their north-star goals, and their perspectives on the businesses they contribute to.
+
+- [ ] While working with individual contributors and during your 1:1s, try to develop an understanding of where and how you want to add value as a manager.
+
+- [ ] Keep a continuous and open discussion going with your peer managers about opportunities for people you think you could manage and why. By the end of sprint rotation we should organically have arrived at a set of suggestions that we, as the entire engineering team, can _consider_.
+
+### Get to know Artsy's tech
+
+- [ ] Unlike individual contributors, who rotate teams for a few sprints to gain a general understanding of how we work as contributors, engineering managers are expected to rotate across _all_ the teams to gain the understanding needed to successfully support their reports in achieving their goals and growth. (For example, unblocking people by knowing how to achieve something or who/where to delegate to, facilitate discussions by understanding context and who to involve for a given topic; and how to set expectations for engineers, product, and business owners as to what is realistically achievable.) Tech-leads and PMs should have been made aware of your main objective and provide you with time to do so.
+
+- [ ] By the end of your sprint rotation, put together a short presentation that summarizes our stack and how different teams contribute to it. (Target audience TBD, although I _think_ it would be good to present it to everyone, both to contribute to presence _and_ so there’s more people that can provide feedback about potential misunderstandings.)
+
+### Get to know Artsy's business
+
+- [ ] Have 1:1s with business owners to make your acquaintance and gain an understanding of what the landscape in their piece of the business looks like in general and how our technology uniquely fuels our take on it.
+
+- [ ] Have 1:1s with PMs to gain an understanding of how product and engineering work together to produce results in their respective parts of the business.
+
+- [ ] TODO: Flesh out: go to a gallery and auction. Besides being fun, what could they learn from doing this and in what way should they do so? E.g. shadow a liaison, casually visit?
```

Comment:
(In my opinion, this TODO doesn’t need to block a merge, we can figure that out together as we go.)